.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "files.exclude": {
+    "**/.git": true,
+    "**/.svn": true,
+    "**/.hg": true,
+    "**/.DS_Store": true,
+    "**/build": true
+  }
+}
```

Comment:
i feel like IDE settings don't belong in repos, and regardless, should be added in a separate PR if they're to be added.